ReactAndroid/src/main/java/com/facebook/react/modules/datepicker/BUCK

```diff
@@ -10,6 +10,7 @@ android_library(
     react_native_target('java/com/facebook/react/bridge:bridge'),
     react_native_target('java/com/facebook/react/common:common'),
     react_native_target('java/com/facebook/react/module/annotations:annotations'),
+    react_native_target('res:shell'),
```

Comment:
Inside ReactAndroid project, The shell resource visibility is in PUBLIC mode. So I have reused that one and added some list of styles to shell resource folder. 

I don't think it is necessary to create a resource folder of each of the package in ReactAndroid project. 

Maybe if the visibility of the shell is in private mode then I would create separate resource folder.

Ref: https://developer.android.com/guide/topics/ui/themes.html